File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/GettingStarted/PythagoreanTriplet.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
    // write your code here  
    Scanner sc= new Scanner(System.in);
    int a=sc.nextInt();
    int b=sc.nextInt();
    int c=sc.nextInt();
    if((a*a==b*b+c*c) || (a*a+b*b==c*c) || (a*a+c*c==b*b))
    System.out.println(true);
    else
    System.out.println(false);
    
   }
  }

File: Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/GettingStarted/PythagoreanTriplet.cpp
#include <iostream>
#include<bits/stdc++.h>
using namespace std;

int main(){
    long long a,b,c;
    cin >> a >> b >> c; 
    
    //write your code here
   int x,y,z;
   x=pow(a,2);
   y=pow(b,2);
   z=pow(c,2);
    if((x==(y+z))||(y==(x+z))||(z==(x+y)))
    {
        cout << "true" ;
    }
    else
        cout << "false" ;
}